packages/aws-library/src/aws_library/ssm/_client.py

```diff
@@ -123,8 +123,11 @@ async def is_instance_connected_to_ssm_server(self, instance_id: str) -> bool:
                 }
             ],
         )
-        assert response["InstanceInformationList"]  # nosec
-        if response["InstanceInformationList"]:
+        if (
+            "InstanceInformationList"  # noqa: RUF019 # the key is actually NOT REQUIRED!
+            in response
```

Comment:
If `response.get` is implemented
```python
if (info := response.get("InstanceInformationList", None) ):
     ...
```

or otherwise using
```python
from toolz import get
if (info := get(response, "InstanceInformationList", None) ):
     ...
```

Reply:
sorry, but the problem is that the model coming from types_aioboto3 is incorrectly setting InstanceInformationList as a required field. alas it is not.